.github/workflows/run-tests.yml

```diff
@@ -257,7 +257,7 @@ jobs:
       - run: mkdir test-results
       - name: Self-evaluation
         id: self-eval
-        run: bin/run.js scanner:run --target ./src --format junit --outfile ./test-results/src.xml --severity-threshold 3
+        run: bin/dev.js scanner run --target ./src --format junit --outfile ./test-results/src.xml --severity-threshold 3
```

Comment:
`bin/dev.js` is the script we should use for running commands in our development environment, now.